Docs: add a few notes and warnings (#2925)
YaLTeR/niri: docs/wiki/Xwayland.md

With no further configuration, niri will create X11 sockets on disk, export `$DISPLAY`, and spawn xwayland-satellite on-demand when an X11 client connects.
If xwayland-satellite dies, niri will automatically restart it.

> [!NOTE]

> > On Void Linux and non Systemd distros, xwayland-satellite will cause niri to crash. To fix this, launch niri with ``dbus-run-session niri --session`` instead.

Reviewer: ```suggestion
> On Void Linux and non-systemd distros, don't forget to launch niri with `dbus-run-session niri --session`.
```
